src/OmniSharp.Abstractions/Endpoints/IGotoDefinition.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+using OmniSharp.Models;
+
+namespace OmniSharp.Endpoints
+{
+    public interface IGotoDefintion
+    {
+        Task<bool> IsApplicableTo(GotoDefinitionRequest request);
```

Comment:
With the right [attributes](https://msdn.microsoft.com/en-us/library/ee155691(v=vs.110).aspx), it might be possible to avoid this method call altogether. Just fetch the gotodefinition handler than can handle C#